source/index.txt

```diff
@@ -32,145 +38,329 @@ Atlas App Services
-   :columns: 2
+Build Data-Driven Apps And Services
+-----------------------------------
 
-   .. card::
-      :headline: Create a New App
-      :url: https://www.mongodb.com/docs/atlas/app-services/manage-apps/create/create-with-ui/
-      :icon: /images/icons/realm/cross-platform.svg
-      :icon-alt: App Icon
-      
-      {+apps+} contain all of your hosted services. Learn how to set up your
-      Atlas cluster and create a new serverless application.
+Services like Device Sync, APIs, and Triggers make it easy to build and
+run virtually any app or backend service on top of Atlas.
 
-   .. card::
-      :headline: Build a Serverless App Backend
-      :url: https://www.mongodb.com/docs/atlas/app-services/tutorial/backend/
-      :icon: /images/icons/realm/realm-lightning.svg
-      :icon-alt: DB Triggers Icon
-      
-      See how to use HTTPS endpoints and triggers to run a serverless GitHub issue
-      tracker.
+.. tabs::
 
-   .. card::
-      :headline: Explore Template Applications
-      :url: https://www.mongodb.com/docs/atlas/app-services/template-apps/
-      :icon: /images/icons/realm/realm-mobile.svg
-      :icon-alt: App Services Mobile Icon
-      
-      Explore working applications to learn how to implement Device Sync in
-      mobile applications or use App Services.
+   .. tab:: Sync Data Across Devices
+      :tabid: device-sync
 
-   .. card::
-      :headline: Learn More About Services
-      :url: https://www.mongodb.com/docs/atlas/app-services/introduction/
-      :icon: /images/icons/realm/graphql.svg
-      :icon-alt: GraphQL Icon
+      .. procedure::
+
+         .. step:: Install a Realm SDK
+
+            Device Sync uses Realm's native language objects to define a
+            data model and work with data.
+            
+            To install an SDK, see the :realm:`Realm SDK documentation
+            </>` for your language and platform.
+
+         .. step:: Enable Sync
+
+            In your Atlas App, specify which fields users can query and
+            define read and write permissions in Dev Mode. Once Sync is
+            turned on, you can start syncing from any app with the Realm
+            SDKs.
+
+            To enable Sync, see :ref:`Enable Atlas Device Sync <enable-sync>`.
+
+         .. step:: Sync Device Data to Atlas
+            
+            Data that you subscribe to in a client app is automatically
+            kept up-to-date. Any changes that the client app makes are
+            automatically synced across all clients whenever a network
+            connection is available.
+            
+      .. image:: /images/plp/realm-sync.png
+         :alt: Realm Sync Illustration
+
+   .. tab:: Access Data with a REST API
+      :tabid: data-api
+
+      .. procedure::
+
+         .. step:: Enable the Data API CRUD & Aggregation Endpoints
+
+            The MongoDB Atlas Data API lets you read and write data in
+            Atlas with standard HTTPS requests. Once enabled, you can
```

Comment:
Nit: I'm not sure we need "standard" here. This made me stop and think "wait, what do they mean 'standard'?" If we can lose "standard" without losing important meaning, I think we should cut it.

Reply:
technically we mean [RFC 2818: HTTPS Over TLS](https://www.rfc-editor.org/rfc/rfc2818) but we can probably just omit it here